tests/integration/cmd_describe_component_test.go

```diff
@@ -67,19 +67,36 @@ var _ = Describe("odo describe component command tests", func() {
 		})
 	}
 
-	It("should fail, with cluster", func() {
+	It("should fail, with default cluster mode", func() {
 		By("running odo describe component -o json with an unknown name", func() {
 			res := helper.Cmd("odo", "describe", "component", "--name", "unknown-name", "-o", "json").ShouldFail()
 			stdout, stderr := res.Out(), res.Err()
 			Expect(helper.IsJSON(stderr)).To(BeTrue())
 			Expect(stdout).To(BeEmpty())
-			helper.JsonPathContentContain(stderr, "message", "no component found with name \"unknown-name\" in the namespace \""+commonVar.Project+"\"")
+			helper.JsonPathContentContain(stderr, "message", "no component found with name \"unknown-name\"")
```

Comment:
I understand the behavior of this message, but I find it a little weird. If the default mode is cluster, shouldn't the namespace part be printed? It would make sense to not print it if we were printing output for both cluster and podman like we do for `odo delete component` command. WDYT? Although this is a case that can be discussed in a different issue.

Reply:
The default mode for `odo describe component` is to try to get info from both platforms, unless specified otherwise.
So yes, the current behavior looks fine to me if Podman is detected when running in the default mode.